.github/workflows/Mythril.yaml

```diff
@@ -11,11 +11,15 @@ jobs:
     strategy:
       matrix:
         contract:
-          - { file: River.1.sol, name: RiverV1}
+          - { file: River.1.sol, name: RiverV1 }
           - { file: Oracle.1.sol, name: OracleV1 }
-          - { file: Withdraw.1.sol, name: WithdrawV1}
-          - { file: Allowlist.1.sol, name: AllowlistV1}
-          - { file: Firewall.sol, name: Firewall}
+          - { file: Withdraw.1.sol, name: WithdrawV1 }
+          - { file: Allowlist.1.sol, name: AllowlistV1 }
+          - { file: Firewall.sol, name: Firewall }
+          - { file: RedeemManager.1.sol, name: RedeemManagerV1 }
+          - { file: ELFeeRecipient.1.sol, name: ELFeeRecipientV1 }
+          - { file: OperatorsRegistry.1.sol, name: OperatorsRegistryV1 }
+          - { file: CoverageFund.1.sol, name: CoverageFundV1 }
```

Comment:
do we want to add in TLC ? And is CoverageFund its own contract that is deployed, if it is why isn't it mentioned in the table on the README page ?

Reply:
cc @iamsahu

Reply:
I had missed TLC an have added it now

Reply:
in which commit ? @iamsahu

Reply:
@mischat : [This](https://github.com/liquid-collective/liquid-collective-protocol/pull/244/commits/8fee059ed1341ada065397d984664cef74387fbc) is the commit

Reply:
great